public/_config.php

```diff
@@ -2,6 +2,8 @@
 session_start();
 require_once("_autoload.php");
 
+$GLOBALS["viewables"]["meta-description"] = "A repoistory of software models imported into Umple.";
```

Comment:
I'm unsure, but is `meta-description` usually application wide or page specific?